sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/objects/objects.scala

```diff
@@ -50,7 +132,7 @@ case class StaticInvoke(
     dataType: DataType,
     functionName: String,
     arguments: Seq[Expression] = Nil,
-    propagateNull: Boolean = true) extends Expression with NonSQLExpression {
+    propagateNull: Boolean = true) extends Expression with InvokeLike with NonSQLExpression {
```

Comment:
why not make `InvokeLike` extends `Expression` and `NonSQLExpression`, then we can just `extends `InvokeLike` here?